src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -196,11 +196,19 @@ const New = () => {
                       as={FileUpload}
                       id="FileUpload-File"
                       name="file"
+                      ariaDescribedBy="FileUpload-Description"
                       onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
                         onChange(e);
                         setFieldValue('file', e.currentTarget?.files?.[0]);
                       }}
                     />
+                    <div
+                      id="FileUpload-Description"
+                      className="sr-only"
+                      tabIndex={-1}
+                    >
+                      Activate to select a file.
```

Comment:
I think the term "Activate" is specific to JAWS. It reads a bit different in VoiceOver and I'd suspect the same for NVDA.

Is "Select a file" not specific enough? I also want to touch on how it's labeled with the word `document`, but we're using `file` here. I want to make sure that's intentional.

Reply:
I'm not sure what's best TBH. JAWS doesn't say it's a file input at all, which is why I used that word, but is it weird to use both terms? On the computer the thing we're talking about is a file, and hitting enter will open the OS dialog that contains files.

In terms of using the word "activate", you might be right that we shouldn't use it.